include/boost/geometry/policies/relate/intersection_points.hpp

```diff
@@ -61,6 +62,21 @@ struct segments_intersection_points
         return result;
     }
 
+    template<typename SegmentIntersectionInfo, typename Point>
+    static inline return_type
+    segments_share_common_point(side_info const&, SegmentIntersectionInfo const& sinfo,
+                                Point const& p)
+    {
+        return_type result;
+        result.count = 1;
+        boost::geometry::assign(result.intersections[0], p);
+
+        // Temporary - this should go later
```

Comment:
When will this go? In this PR?

Reply:
This is copy paste from the segments_crosses() function above. I do not understand it either, but without that line the results are not correct, thus the information assigned to fractions is useful and cannot go now.

Reply:
Do you have any idea of the reasoning behind the original comment?

Reply:
Actually, the whole fractions could be revised / simplified if rescaling goes.
It might be that the comment is related to that.